src/schema/launchpadSchemas.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from "zod";
+
+export const getAllLaunchpadsSchema = z.object({
+  status: z.enum(["active", "retired", "under construction"]).optional(),
+});
+export type getAllLaunchpadsSchema = z.infer<typeof getAllLaunchpadsSchema>;
+
+export const latLngSchema = z.object({
```

Comment:
И оваа шема јас би ја преименувал според ендпоинт-о
```suggestion
export const getClosestSchema = z.object({
```